src/main/java/com/nirima/jenkins/plugins/docker/builder/DockerBuilderPublisher.java

```diff
@@ -133,6 +135,7 @@
     public final String cloud;
     public /* almost final */ boolean noCache;
     public /* almost final */ boolean pull;
+    public Map<String, String> labels;
```

Comment:
See [comment below](https://github.com/jenkinsci/docker-plugin/pull/769#discussion_r368930828) - I think it'd be better to store `labelString` rather than the parsed result of the string the user specified.
If we only store the parsed result, we can't store an invalid result, meaning that anything the user types in that we can't parse gets destroyed rather than letting the user go back and re-edit it.

Reply:
..and, also, it should be `/* almost final */` too, because it's only set by the setter at construction time - it's not really something that's mutated while the code is running.